doc/API.md

```diff
@@ -27,13 +27,114 @@ Example GET curl call:  `curl -A "Sia-Agent" /wallet/transactions?startheight=1&
 
 Example POST curl call: `curl -A "Sia-Agent" --data "amount=123&destination=abcd" /wallet/siacoins
 
+Daemon
+------
+
+Queries:
+
+* /daemon/constants [GET]
+* /daemon/stop      [GET]
+* /daemon/version   [GET]
+
+#### /daemon/constants [GET]
```

Comment:
I was thinking we could have a console or an api plugin, where you can type siac commands or input api calls and get back raw output.